File: vijos/1298.py
T=int(input())
for Testcase in range(T):
    alen=int(input())
    a=[int(i) for i in input().split()]
    if len(a)%2==1:
        print("lolanv")
    else:
        a.sort()
        cnt=0
        for i in range(0,alen-1,2):
            if a[i]==a[i+1]:
                cnt+=1
        if cnt==alen/2:
            print("Wind")
        else:
            print("lolanv")

File: vijos/1298.cpp
#include <cstdio>
#include <algorithm>
int a[10005];
int main()
{
	int T;scanf("%d",&T);
	while(T--)
	{
		int n;scanf("%d",&n);
		for(int i=1;i<=n;i++)
			scanf("%d",&a[i]);
		if(n&1){printf("lolanv\n");continue;}
		std::sort(a+1,a+1+n);
		int cnt=0;
		for(int i=1;i<=n;i+=2)
			cnt+= a[i]==a[i+1];
		printf("%s\n",cnt==n/2? "Wind":"lolanv");
	}
}